packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewDefault.html

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="iframe-wrapper iframe-wrapper__default">
+    <div
+      onclick={stopPropagation}
+      class="quickview__spinner-container slds-is-relative wrapper"
+    >
+      <lightning-spinner
+        if:true={isLoading}
+        alternative-text="Loading"
+        size="large"
+      ></lightning-spinner>
+      <iframe
+        id="quickview__content-iframe"
+        class="quickview__content-container slds-modal__content slds-p-around_large slds-wrap"
+        src={contentURI}
+        height="100%"
+        width="100%"
```

Comment:
yeah just read a bit on this, it seems better to include in css rather than the iframe tag properties.

More flexible, better for accessibility and avoid certain issues with putting them directly in the tag (ie performance loading issues, display issues)